var/spack/repos/builtin/packages/py-radical-entk/package.py

```diff
@@ -11,89 +11,36 @@ class PyRadicalEntk(PythonPackage):
 
     homepage = "https://radical-cybertools.github.io"
     git = "https://github.com/radical-cybertools/radical.entk.git"
-    pypi = "radical.entk/radical.entk-1.47.0.tar.gz"
+    pypi = "radical_entk/radical_entk-1.90.0.tar.gz"
 
     maintainers("andre-merzky")
 
     license("MIT")
 
     version("develop", branch="devel")
-    version("1.47.0", sha256="a4338e3a87147c032fb3a16a03990155742cc64c6625cfb4e1588ae0e51aafda")
-    version("1.39.0", sha256="72d64b25df9f3cb1dcbc32323a669d86d947cf07d15bed91cfedca2a99fb3ef1")
+    version("1.90.0", sha256="09430e9a337cd41d7499357bdbfb01e20679d59e83ae15d5799de2f8eb530a45")
 
     version(
-        "1.20.0",
-        sha256="1b9fc470b926a93528fd2a898636bdcd1c565bd58ba47608f9bead811d8a46d7",
+        "1.47.0",
```

Comment:
Is 1.47.0 still downloadable with the new pypi attribute? You may need a url_for_version function.

Reply:
you are right, `url_for_version` is needed. Updated accordingly